src/content/docs/commands/PING.md

```diff
@@ -22,58 +16,84 @@ PING [message]
+
+## Errors
 
-The return value varies based on the presence of the optional `[message]` parameter.
+1. `Syntax Error`: If the syntax is incorrect, such as including unexpected additional parameters, an error will be raised:
 
-- `No message provided`: If no message is provided, `PING` will return a simple string reply `PONG`.
-- `Message provided`: If a message is provided, `PING` will return a simple string that echoes the given message.
+   - `Error message`: `(error) ERR wrong number of arguments for 'ping' command`
+   - `Scenario`: If more than one argument is provided.
+
+   ```bash
+   127.0.0.1:7379> PING "Message 1" "Message 2"
+   (error) ERR wrong number of arguments for 'ping' command
+   ```
 
 ## Example Usage
 
-### Example 1: Pinging the server without a message
+### Basic Usage 
+
+`PING` DiceDB server without a message and the server echoes with `PONG`.
 
 ```bash
 127.0.0.1:7379> PING
 PONG
 ```
 
-### Example 2: Pinging the server with a message
+### Pinging the server with a message
+
+DiceDB server is pinged with `Hello, DiceDB!` and the server echoes with `Hello, DiceDB!`.
 
 ```bash
 127.0.0.1:7379>  PING "Hello, DiceDB!"
 "Hello, DiceDB!"
 ```
 
-## Behaviour
-
-When the `PING` command is fired:
+### Pinging the server with int message
 
-1. If no message is given, it sends back a `PONG`.
-1. If a message is provided, it sends back the same message in the response.
+DiceDB server is pinged with `1234` and the server echoes with `"1234"` coerced to string internally.
 
-This helps clients determine if the server is up and responsive. Essentially, it acts as a keep-alive or heartbeat mechanism for clients to validate their connection with the DiceDB server.
-
-## Error Handling
-
-### Potential Errors
-
-1. `Syntax Error`: If the syntax is incorrect, such as including unexpected additional parameters, an error will be raised:
+```bash
+127.0.0.1:7379>  PING 1234
+"1234"
+```
 
-   - `Error message`: `(error) ERR wrong number of arguments for 'ping' command`
-   - `Scenario`: If more than one argument is provided.
+### Pinging the server with list message
 
-   ```bash
-   > PING "Message 1" "Message 2"
-   (error) ERR wrong number of arguments for 'ping' command
-   ```
+DiceDB server is pinged with `[1234]` and the server echoes with `"[1234]"` coerced to string internally.
 
-2. `Data Type Error`: If the argument provided is not a string (for example, if a list or other data type is provided), a type error will be raised.
+```bash
+127.0.0.1:7379>  PING [1234]
+"[1234]"
+```
 
-   - `Error message`: This specific type of error handling is internal, and improper types are generally implicitly converted or rejected by the protocol, raising basic syntax errors or allowing them based on DiceDB' internal type coercion rules.
+### Invalid Usage
 
-## Additional Notes
```

Comment:
reviewer-note: removed Additional Notes section as it was not present in the #38. Do let me know if you want me to add it back.